source/components/Image/index.js

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+// import Vignette from 'vignette';
+
+/**
+ * Create a super low resolution image that will automatically be blurred in most browsers
+ *
+ * @param vignetteUrl
+ * @returns {String}
+ */
+function getLowRes(vignetteUrl) {
+    return vignetteUrl;
+    // return Vignette.getThumbURL(vignetteUrl, {
+    //     mode: 'scale-to-width-down',
+    //     width: 5,
+    //     height: 5,
+    // });
+}
+
+class Image extends React.Component {
+    constructor(props) {
+        super(props);
+        const { src } = props;
+        this.onLoad = this.onLoad.bind(this);
+        this.state = {
+            src,
+            loading: true,
+            limbo: false,
+        };
+    }
+
+    // When the src changes first replace the src with a temp image so it doesn't stall displaying
+    // the old image
+    // https://reactjs.org/blog/2018/06/07/you-probably-dont-need-derived-state.html#when-to-use-derived-state
+    static getDerivedStateFromProps(props, state) {
```

Comment:
I do not think the setting state in the constructor is necessary with this method.